databases/week1/lesson1.sql

```diff
@@ -0,0 +1,20 @@
+SELECT * FROM task;
+-- Find out how many tasks are in the task table --
+SELECT COUNT(id)FROM task
+WHERE due_date IS NULL;
+-- Find all the tasks that are marked as done --
+SELECT * FROM task
+WHERE status_id = 3;
+SELECT * FROM status;
+SELECT * FROM task
+WHERE status_id IS NULL;
+-- Get the title and due date of all tasks where the title or description contains database --
+SELECT * FROM task
+WHERE description or title LIKE '%database%';
```

Comment:
After WHERE clause we define what we need, for example `WHERE data LIKE '%something%' OR data2 LIKE '%something%'`